content/articles/2019-08-gargle-hello-world.Rmarkdown

```diff
@@ -0,0 +1,88 @@
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+## Introduction
+
+We're elated to announce the initial release of gargle and of googledrive v1.0.0 on CRAN!
+
+  * The [gargle package](https://gargle.r-lib.org) is meant to take some of the pain out of working with Google APIs. It's mostly aimed at the maintainers of R packages that call Google APIs. If we're successful, users won't really notice gargle, they'll just feel like these packages got easier to work with and less idiosyncratic.
+  * The [googledrive package](https://googledrive.tidyverse.org) is a great example of such a package: it helps useRs work with Google Drive from R. The v1.0.0 release gets all auth functionality (and more) from gargle and also has a new feature that makes file upload/update more user-friendly.
+  
+This post focuses on gargle, but don't forget to check out the companion post describing googledrive v1.0.0. The bigrquery package also uses gargle as of v1.2.0, which you can read about in its [changelog](https://bigrquery.r-dbi.org/news/index.html#bigrquery-1-2-0).
+  
+## gargle
+
+The [gargle package](https://gargle.r-lib.org) first appeared on CRAN in early June 2019 and has gotten a few small updates as we learn more by wiring it into packages. gargle is already used in these packages:
+
+  * [bigrquery](https://bigrquery.r-dbi.org) (>= v1.2.0)
+  * [googledrive](https://googledrive.tidyverse.org) (>= v1.0.0)
+  * [gmailr](https://gmailr.r-lib.org) (>= v1.0.0 *coming soon to CRAN*)
+  * [googleAuthR](https://code.markedmondson.me/googleAuthR/) (*dev version*)
+  * [googlesheets4](https://googlesheets4.tidyverse.org) *GitHub only*
+  * [gcalendr](https://andrie.github.io/gcalendr/) *GitHub only*
+
+gargle's current functionality falls into two domains, which a client package can adopt (or not) separately:
+
+  * Auth: help users authenticate themselves with their Google identity and
+    obtain a token that the wrapper package can use to make authorized requests.
+  * Request preparation and response handling: check requests against the
+    machine-readable [Discovery Documents](https://developers.google.com/discovery/v1/reference/apis)
+    that describe Google APIs and process API responses, especially errors.
+    
+The long-term stretch goal is to do for R what the [official Google API Client Libraries](https://developers.google.com/api-client-library/) do for other languages, like Python and Java.
+
+## Auth via gargle
+
+Under the hood, gargle's main auth function is [`token_fetch()`](https://gargle.r-lib.org/articles/how-gargle-gets-tokens.html), which tries a series of different methods for obtaining a token. The intent is to make auth "just work" in a wide variety of contexts. This makes some flows newly available in the client packages, such as [Application Default Credentials](https://www.jhanley.com/google-cloud-application-default-credentials/).
```

Comment:
FWIW, https://cloud.google.com/docs/authentication/production is the "official" link for this, but is less readable than the page you linked; worth including both?

Reply:
Added.